brainglobe_atlasapi/atlas_generation/atlas_scripts/humanatlas.py

```diff
@@ -1,27 +1,42 @@
-import gzip
+### Import
 import json
 import multiprocessing as mp
 import time
-import zipfile
-from pathlib import Path
 
 import numpy as np
 import pandas as pd
-import requests
+import pooch
 import treelib
 import urllib3
 from allensdk.core.structure_tree import StructureTree
 from brainglobe_utils.image_io import load_nii
 from rich.progress import track
 
+from brainglobe_atlasapi import utils
 from brainglobe_atlasapi.atlas_generation.mesh_utils import (
     Region,
     create_region_mesh,
     inspect_meshes_folder,
 )
 from brainglobe_atlasapi.atlas_generation.wrapup import wrapup_atlas_from_data
+from brainglobe_atlasapi.config import DEFAULT_WORKDIR
 from brainglobe_atlasapi.structure_tree_util import get_structures_tree
 
+### Metadata
+__version__ = "0"
+
+RES_UM = 500
+VERSION = 1
+ATLAS_NAME = "allen_human"
+SPECIES = "Homo sapiens"
+ATLAS_LINK = "http://download.alleninstitute.org/informatics-archive/allen_human_reference_atlas_3d_2020/version_1/"
+CITATION = "Ding et al 2016, https://doi.org/10.1002/cne.24080"
+ORIENTATION = "rpi"
+
+### Settings
+PARALLEL = True  # disable parallel mesh extraction for easier debugging
```

Comment:
Does parallelisation work for this atlas? It's probably not good to have it as a default if not... and recently we've been moving away from parallelisation here?

Reply:
Weirdly, it does, if I remember correctly. I agree though, it shouldn't be the default setting.